cfpb/consumerfinance.gov
# Replace jQuery in "Before You Claim"

This PR replaces jquery functions and methods with "vanilla" JavaScript so we no longer rely on jQuery in this app.

I attempted to replace jQuery without changing any lines of the existing code, but I did have to change the reference to a few event handlers just for ease. The existing functional tests are passing.

## Additions
- Add `dollar-sign.js` which replaces relevant jQuery methods.

## Changes
- Change a few event handlers to use `Query.on()` 


## How to test this PR
1. Check the functional tests in cypress
2. Visually check [the app](http://localhost:8000/consumer-tools/retirement/before-you-claim/) to make sure it matches the [production apa](http://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/)

## Checklist

### Front-end testing

*This app should function exactly as it did before.*
